Tests/Controller/HelperControllerTest.php

```diff
@@ -328,6 +329,20 @@ public function testappendFormFieldElementAction($validatorInterface)
 
         $twig = new \Twig_Environment($this->getMock('\Twig_LoaderInterface'));
         $twig->addExtension(new FormExtension($mockRenderer));
+
+        if (Kernel::MAJOR_VERSION >= 3 && Kernel::MINOR_VERSION >= 2) {
```

Comment:
We shouldn't. I'm currently waiting for the tests to pass, and then I think we should do a complete review, but yeah this is not the way to go. Detecting the presence of the public property might be a solution.